src/main/resources/jenkins/plugins/extracolumns/BuildDurationColumn/columnHeader.jelly

```diff
@@ -31,5 +31,8 @@ THE SOFTWARE.
         <j:when test="${it.getBuildDurationType()==1}">
             <th>${%Average Duration}</th>
         </j:when>
+        <j:when test="${it.getBuildDurationType()==2}">
+            <th title="${%Average duration in minutes (compact)}">⏳'</th>
```

Comment:
@fredg02 I am really curious what makes older Jenkins incompatible with Unicode Emojis? As far as I know is only Unicode.